README.md

```diff
@@ -1,5 +1,59 @@
-# curriculum-cr-training-contact-form
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+# My Portofolio
+> this porject is my portofolio an ultimate way to showcase your work and let others reach out for me.\
+![screenshot](./media/png/ScreenShot.png)\
+> It presents my professionalism and dedication to the craft better and more clearly than any CV.\
+
+
+## Built With
+
+- HTML & CSS
+- Html & Css linters
+
+## Live Demo
+
+### [LIVE DEMO LINK](https://aimemalaika.github.io/portofolio)
+
+
+## Getting Started
+- clone the repository by running\
+    `git clone https://github.com/aimemalaika/portofolio.git`
+- navigate to the folder\
+    `cd portofolio`
+
+## Authors
+
+👤 **Aime Malaika**
+
+- GitHub: [@aimemalaika](https://github.com/aimemalaika)
+- Twitter: [@Aime_malaika](https://twitter.com/Aime_Malaika)
+- LinkedIn: [aimemalaika](https://linkedin.com/in/aimemalaika)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
[OPTIONAL] README is almost ready, please add a link to the repository's issues page, here's an [example.](https://github.com/microverseinc/readme-template/blob/master/README.md)